### PR TITLE
[docs] Improve reference index

### PR DESCRIPTION
**Pull request by brentvatne**
# Why

ENG-9768 + some small tweaks while in this file

### CODE REVIEW

File: docs/pages/versions/unversioned/index.mdx [hunk 1]
```diff
@@ -50,6 +49,19 @@ The easiest way to create a bare React Native app with support for the Expo SDK
...
 />
 
+## Using pre-release versions
+
+New Expo SDK versions are released three times each year. In between releases, we publish pre-release versions of the `expo` package and all of the Expo SDK packages. Pre-releases are not considered to be stable and should only be used if you are comfortable with the risk of encountering bugs or other issues.
+
+Pre-release versions of the `expo` package are versioned with an `-alpha` suffix, such as `50.0.0-alpha.1`. To install, for example, SDK 50 alpha, you can run the following command:
+
+<Terminal
+  cmd={['# Install the alpha version of expo and its related packages', '$ npm install expo@50.0.0-alpha && npx expo install --fix']}
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/index.mdx:59]: we will want to update this once we come up with a better process and having tooling that supports that. for example, we should probably tag alpha releases with the `alpha` tag on npm. cc @wschurman

File: docs/pages/versions/unversioned/index.mdx [hunk 2]
```diff
@@ -24,22 +24,21 @@ import { Gyroscope } from 'expo-sensors';
 
 This allows you to write [`Contacts.getContactsAsync()`](/versions/latest/sdk/contacts#contactsgetcontactsasynccontactquery) and read the contacts from the device, read the gyroscope sensor to detect device movement, or start the phone's camera and take photos.
 
-## These packages work in bare React Native apps too
+## All Expo SDK packages work in any React Native app
 
-The easiest way to create a bare React Native app with support for the Expo SDK is by running the command:
+...and of course they do, because Expo apps are React Native apps! The easiest way to create a React Native app with support for Expo SDK packages is to use `create-expo-app`, but you can also add Expo SDK support to an existing React Native app with the `npx install-expo-modules` command.
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/index.mdx:29]:
```suggestion
Expo apps are React Native apps! The easiest way to create a React Native app with support for Expo SDK packages is to use `create-expo-app`. However, you can also add Expo SDK support to an existing React Native app with the `npx install-expo-modules` command.
```

Please, let's not use ellipses.

File: docs/pages/versions/unversioned/index.mdx [hunk 3]
```diff
@@ -24,22 +24,21 @@ import { Gyroscope } from 'expo-sensors';
...
 
 <Terminal
   cmd={['# Create a project named my-app', '$ npx create-expo-app my-app --template bare-minimum']}
   cmdCopy="npx create-expo-app my-app --template bare-minimum"
 />
 
 <BoxLink
-  title="Existing apps"
+  title="Installing Expo SDK packages in existing React Native apps"
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/index.mdx:37]:
```suggestion
  title="Install Expo SDK packages in existing React Native apps"
```

File: docs/pages/versions/unversioned/index.mdx [hunk 4]
```diff
@@ -50,6 +49,19 @@ The easiest way to create a bare React Native app with support for the Expo SDK
   href="/workflow/using-libraries"
 />
 
+## Using pre-release versions
+
+New Expo SDK versions are released three times each year. In between releases, we publish pre-release versions of the `expo` package and all of the Expo SDK packages. Pre-releases are not considered to be stable and should only be used if you are comfortable with the risk of encountering bugs or other issues.
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/index.mdx:54]:
```suggestion
New Expo SDK versions are released three times each year. Between these releases, we publish pre-release versions of the `expo` package and all of the Expo SDK packages. Pre-releases are not considered stable and should only be used if you are comfortable with the risk of encountering bugs or other issues.
```

File: docs/pages/versions/unversioned/index.mdx [hunk 5]
```diff
@@ -50,6 +49,19 @@ The easiest way to create a bare React Native app with support for the Expo SDK
...
+
+Pre-release versions of the `expo` package are versioned with an `-alpha` suffix, such as `50.0.0-alpha.1`. To install, for example, SDK 50 alpha, you can run the following command:
+
+<Terminal
+  cmd={['# Install the alpha version of expo and its related packages', '$ npm install expo@50.0.0-alpha && npx expo install --fix']}
+  cmdCopy="npm install expo@50.0.0-alpha && npx expo install --fix"
+/>
+
+You can often use pre-release versions of individual packages with stable releases of the Expo SDK, but there may occasionally be incompatibilities or other issues that arise in alpha-quality releases. You may want to [silence dependency validation warnings](https://docs.expo.dev/more/expo-cli/#configuring-dependency-validation) if opt in to alpha package, once you verify that it works well for your use cases.
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/index.mdx:63]:
```suggestion
You can often use pre-release versions of individual packages with stable releases of the Expo SDK. There may occasionally be incompatibilities or other issues that arise in alpha-quality releases. You may want to [silence dependency validation warnings](/more/expo-cli/#configuring-dependency-validation) if you opt in to the alpha package and once you have verified that it works well for your use cases.
```

**Review by amandeepmittal — COMMENTED** [review]
Left some suggestions to improve the phrasing for better readability.

File: docs/pages/versions/unversioned/index.mdx (hunk 2 above)
Comment (brentvatne) [reply docs/pages/versions/unversioned/index.mdx:29]: can you elaborate? I don't see any mention of this in the docs style guide so I'm curious about the reasoning

**Pull request merged**